.gitignore

```diff
@@ -10,3 +10,4 @@ __pycache__
 ansible.cfg
 pyratlabs-issue-dump.txt
 .cache
+/.idea/
```

Comment:
What's this for?

Reply:
work ide env is idea+OrchidE plugin, git clone and import to my workspace will auto create .idea dir.